lib/Extractor/KLEEBuilder.cpp

```diff
@@ -143,14 +149,26 @@ ref<Expr> ExprBuilder::makeSizedArrayRead(unsigned Width, StringRef Name,
   std::vector<unsigned> ZeroBits, OneBits;
   UpdateList UL(Arrays.back().get(), 0);
   ref<Expr> Var = ReadExpr::create(UL, klee::ConstantExpr::alloc(0, Expr::Int32));
-  if (Origin && Origin->K == Inst::Var && (Origin->KnownZeros.getBoolValue() ||
-                                           Origin->KnownOnes.getBoolValue())) {
```

Comment:
I think you meant the OR expression to look like as its in line : 153 below. The line is deleted for which you pointed a change. (?)